packages/eas-cli/src/commandUtils/EasCommand.ts

```diff
@@ -52,7 +101,18 @@ export default abstract class EasCommand extends Command {
       // commands/submit === submit, commands/build/list === build:list
       action: `eas ${this.id}`,
     });
-    return this.runAsync();
+
+    const jsonOutput = await this.runAsync({
```

Comment:
I like the pattern, it enforces that we only print one json object, but this might require significant refactor in some more complex commands.

Reply:
Yep!